lib/oli_web/plugs/set_user.ex

```diff
@@ -21,13 +22,22 @@ defmodule Oli.Plugs.SetCurrentUser do
     if author = Pow.Plug.current_user(conn, pow_config) do
       cond do
         current_author = Repo.get(Author, author.id) ->
+          is_community_admin =
+            Accounts.list_admin_communities(current_author.id)
+            |> Enum.empty?() ==
```

Comment:
Either pipe current_author.id into list_admin_communities, or change to Enum.empty?(Accounts.list_admin_communities(current_author.id)) - pipe preferred. Elixir style guide is pretty clear about not using pipe with only one item.